src/components/cards/SwapCard/SwapCard.vue

```diff
@@ -365,15 +365,23 @@ export default defineComponent({
         dismissedErrors.value.highPriceImpact = true;
       }
     }
+
+    function isNativeAssetIdentifier(assetParam: string): boolean {
+      return (
+        assetParam?.toLowerCase() === nativeAsset.deeplinkId?.toLowerCase() ||
```

Comment:
we should specify why we use optional chaining here `assetParam?.` in arg type:
e.g `(assetParam: string | undefined)`

Reply:
Good point, fixed.